RpcNep5Balances/RpcNep5Balances.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>2.9.5</Version>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <RootNamespace>Neo.Plugins</RootNamespace>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Update="RpcNep5Balances\config.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Neo" Version="2.9.5" />
```

Comment:
This depends on some recent changes to Neo expected to be in the 2.9.5 release (Specifically https://github.com/neo-project/neo/pull/613).